runtime/vm/CMakeLists.txt

```diff
@@ -198,15 +208,24 @@ if(interp_new_flags)
 endif()
 
 
-# JIT helper methods require us to disable buffer security checks on Windows
-# See https://github.com/eclipse/openj9/pull/1494 for rationale
+
 if(OMR_OS_WINDOWS AND (OMR_TOOLCONFIG STREQUAL "msvc"))
+	# JIT helper methods require us to disable buffer security checks on Windows
+	# See https://github.com/eclipse/openj9/pull/1494 for rationale
 	set_property(
 		SOURCE
 			cnathelp.cpp
 			SharedService.c
 		APPEND PROPERTY COMPILE_FLAGS "/GS-"
 	)
+
+	# Use J9VM_C_ALT compiler (aka clang) to build the interpreter
+	enable_language(J9VM_C_ALT)
```

Comment:
I think this should be based on a better name for the language; we're talking about C++, not C: I suggest `J9VM_CXX` - I don't think we need the '`_ALT`' suffix, or we could replace the '`J9VM_`' prefix with '`ALT_`' to get `ALT_CXX`.